### ISSUE TITLE
Episodes do not download because season is detected as complete

### ISSUE THREAD

**Issue by focher** [Sonarr/Sonarr#7446]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Current Behavior

Currently, Sonarr detects an entire season as downloaded even when it is mid-season and then does not download individual episodes. This behavior would be normal if the season is complete, but now some torrent offerings are combining all episodes to date in a season in a way that makes Sonarr thinks the entire season is finished. See https://github.com/Sonarr/Sonarr/issues/6987.

### Expected Behavior

Episodes that are new should be downloaded. Unfortunately, some torrents are being offered that suggest the season is complete even though it's still mid-season so only episodes available to-date are offered in a single new torrent. This is confusing Sonarr into thinking that all episodes - even new ones - are available and already downloaded.

### Steps To Reproduce

1. Subscribe to a new TV series.
2. Wait for a TV series torrent to be offered that is "season based" even though the season has not fully been aired.
3.  Sonarr detects a season torrent with only the currently aired episodes in mid-season.
4.  Sonarr does not detect single subsequent episodes because it believes it already has downloaded a full season of the series.

### Environment

```markdown
- OS: Docker
- Sonarr: Latest
- Docker Install: Latest 
- Using Reverse Proxy: No
- Browser: Any
- Database: Docker contained
```

### What branch are you running?

Main

### Trace Logs?

Available if this issue is accepted.

**Comment by augustuen** [comment]
"Latest" isn't a version. Especially with Docker setups, an instance can be fully "updated", yet be months behind the actual development.

And I think you're gonna have to produce some concrete examples to show what you're talking about. With screenshots.
